### ISSUE TITLE
How to run noqa on a YAML literal block

### ISSUE THREAD

**Issue by ianw** [ansible/ansible-lint#534]
# Issue Type
- Bug report
- Feature request(?)

# Desired Behaviour

Given something like

```yaml
exim_system_aliases_router: |
  driver = redirect
  allow_fail
  allow_defer
  data = ${lookup{$local_part}lsearch{/etc/aliases}}
  file_transport = address_file
  pipe_transport = address_pipe
```

This trips 

```
[206] Variables should have spaces before and after: {{ var_name }}
```

I'm at a loss how to actually skip this with noqa?  I don't want a ```#noqa XXX``` comment on the end of the line, because it will go into the string.  I tried

```yaml
exim_system_aliases_router: # noqa 206
 |
  driver = redirect
  allow_fail
  allow_defer
  data = ${lookup{$local_part}lsearch{/etc/aliases}}
  file_transport = address_file
  pipe_transport = address_pipe
```

which is ugly ... but also didn't work.

Happy to update documentation if I can figure it out ...

**Comment by awcrosby** [comment]
The way ansible-lint analyses line-based rules (like 206) is by running over each line in the file text, not by parsing beforehand (like task-based rules). So unfortunately a `# noqa` that is not present on the specific line being analyzed will not work.

However I think we can cover this particular false positive by ensuring at least `{{` and `}}` both exist before applying further regex - raised PR #535

**Comment by eigengrau** [comment]
Cross-reference #586
